content/en/opentelemetry/setup/otlp_ingest_in_the_agent.md

```diff
@@ -104,7 +104,7 @@ OTLP logs ingestion on the Datadog Agent is disabled by default so that you don'
    - Set `DD_OTLP_CONFIG_LOGS_ENABLED` to true.
 
 <div class="alert alert-warning">
-<strong>Known Issue</strong>: Agent versions 7.61.0 through 7.63.0 have an issue where OTLP ingestion pipelines may fail to start in Docker environments, showing the error: <code>Error running the OTLP ingest pipeline: failed to register process metrics: process does not exist</code>.<br>
+<strong>Known Issue</strong>: Starting from Agent versions 7.61.0, the OTLP ingestion pipelines may fail to start in Docker environments showing the error: <code>Error running the OTLP ingest pipeline: failed to register process metrics: process does not exist</code>.<br>
```

Comment:
```suggestion
<strong>Known Issue</strong>: Starting with Agent version 7.61.0, OTLP ingestion pipelines may fail to start in Docker environments, displaying the error: <code>Error running the OTLP ingest pipeline: failed to register process metrics: process does not exist</code>.<br><br>
```

Reply:
The additional `<br>` tags should help with the spacing. I'd recommend adding one more `<br>` tag after step 3. (GitHub won't let me add it as a suggestion.)